src/AWS.Deploy.Common/Recommendation.cs

```diff
@@ -200,10 +200,35 @@ public bool IsOptionSettingDisplayable(OptionSettingItem optionSetting)
                 var dependsOnOptionSettingValue = GetOptionSettingValue(dependsOnOptionSetting);
                 if (
                     dependsOnOptionSetting != null &&
-                    dependsOnOptionSettingValue != null &&
-                    !dependsOnOptionSettingValue.Equals(dependency.Value))
+                    dependsOnOptionSettingValue != null)
                 {
-                    return false;
+                    if (dependency.Operation == null ||
+                           dependency.Operation == PropertyDependencyOperationType.Equals)
+                    {
+                        if (!dependsOnOptionSettingValue.Equals(dependency.Value))

```

Comment:
Can you pass in the InvaritantCulture for comparison?

Reply:
both `dependsOnOptionSettingValue` and `dependency.Value` are objects not strings and i don't want to cast them to string to use StringComparison.InvariantCulture